kover-gradle-plugin/docs/aggregated.md

```diff
@@ -0,0 +1,67 @@
+# Kover Aggregated Plugin
+
+## Quickstart
+This plugin is a prototype for testing ideas related to an alternative configuration method.
+
+The main difference from the existing Kover Gradle Plugin is the reactive content of the report: only those classes and tests that were compiled and executed within the same build get into the report.
+
+To use the plugin, just add into a `settings.gradle.kts` file 
+```kotlin
+plugins {
+    id("org.jetbrains.kotlinx.kover.settings") version "0.8.1"
```

Comment:
I'm not sure that `kover.settings` conveys the meaning of the plugin. Yes, it is applied in settings, but its goals are 1) reduce configuration 2) instrument only test tasks that are currently run. We may want to eventually rename it, but I don't have any good name right now except `kover.onthefly`

Reply:
Given that we have `merge` in regular setup which also does some kind of aggregation and various `AggregationType`s, I'm afraid that this word is already overused :(

Reply:
In fact, this plugin has nothing to do with old Kover plugin, but it does exactly what `merge` says - it creates a aggrgated reports

Reply:
Well, if I run `./gradlew :projectA:jvmTest -Pkover koverHtmlReport`, it will show me only projectA's coverage, in that case there's no aggregation

Reply:
Not quite. `:projectA` may depends on `:projectB`, thus the sources from these two projects will be aggregated in the report

Reply:
I can live with `aggregation` for now, maybe we'll find better name later